src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandlerTest.java

```diff
@@ -94,6 +82,24 @@ void shouldReturnWithoutError() {
         assertThat(response.getErrors()).isEmpty();
     }
 
+    @Test
+    void givenPreviewDocumentHasBeenSetAndCorrectionsRefused_thenAddToDocumentsTab() {
+        when(callback.getCaseDetails()).thenReturn(caseDetails);
+        when(caseDetails.getCaseData()).thenReturn(caseData);
+
+        caseData.getDocumentStaging().setPreviewDocument(DocumentLink.builder().documentUrl("document url test").build());
+        caseData.getPostHearing().getCorrection().setAction(CorrectionActions.REFUSE);
+
+        PreSubmitCallbackResponse<SscsCaseData> response =
+            handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
+
+        assertThat(response.getErrors()).isEmpty();
+
+        List<SscsDocument> documents = response.getData().getSscsDocument();
+        assertThat(documents).hasSize(1);
+        assertThat(documents.get(0).getValue().getDocumentType()).isEqualTo(DocumentType.CORRECTION_REFUSED.getValue());
```

Comment:
extracting can be used to chain assertions here
```suggestion
assertThat(documents)
    .hasSize(1)
    .element(0)
    .extracting(doc -> doc.getValue().getDocumentType())
    .isEqualTo(DocumentType.CORRECTION_REFUSED.getValue());
```